<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarketReportDialog</class>
 <widget class="QDialog" name="MarketReportDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>653</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Generate Market Report</string>
  </property>
  <widget class="QLineEdit" name="minHead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minPrice">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Min. Price</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Min. Head</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="suppressSellerBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>610</y>
     <width>290</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Suppress Seller Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>610</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="printButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>610</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>760</height>
    </size>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRound5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Round to $0.05?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="colorCombo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0-69</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70-79</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80-89</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90-99</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>All</string>
    </property>
   </item>
  </widget>
  <widget class="Q3GroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>800</width>
     <height>250</height>
    </rect>
   </property>
   <property name="title">
    <string>Included List</string>
   </property>
   <widget class="QLabel" name="textLabel2_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter ID To Exclude:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="idExcludeFilter">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel2_2_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Then Press Enter</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="excludePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="excludeAllPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Q3Table" name="includeTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>780</width>
      <height>180</height>
     </rect>
    </property>
    <property name="numRows">
     <number>0</number>
    </property>
    <property name="numCols">
     <number>9</number>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="rowMovingEnabled">
     <bool>false</bool>
    </property>
    <property name="sorting">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>Q3Table::MultiRow</enum>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Seller</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Head</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Avg. Weight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price/CWT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price/Head</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Amount</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="applyFiltersButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Reload List With Filters</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>800</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string>Transaction List</string>
   </property>
   <widget class="QLineEdit" name="idIncludeFilter">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter ID To Include:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel2_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>200</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Then Press Enter</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="includePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="includeAllPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Q3Table" name="transactionTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>780</width>
      <height>170</height>
     </rect>
    </property>
    <property name="numRows">
     <number>0</number>
    </property>
    <property name="numCols">
     <number>9</number>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="rowMovingEnabled">
     <bool>false</bool>
    </property>
    <property name="sorting">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>Q3Table::MultiRow</enum>
    </property>
    <property name="focusStyle">
     <enum>Q3Table::FollowStyle</enum>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Seller</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Head</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Avg. Weight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price/CWT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price/Head</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Amount</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QLabel" name="textLabel1_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Index By:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="indexComboBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Seller</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Head</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Description</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Avg. Weight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Price/CWT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Price/Head</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Amount</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Id</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">dao/transactiondao.h</include>
  <include location="local">reports/marketreport.h</include>
  <include location="global">algorithm</include>
 </includes>
 <resources/>
 <connections/>
</ui>
